src/common/libjob/test/job.c

```diff
@@ -115,46 +115,6 @@ void check_corner_case (void)
     ok (flux_job_submit_get_id (NULL, NULL) < 0 && errno == EINVAL,
         "flux_job_submit_get_id with NULL args fails with EINVAL");
 
-    /* flux_job_list */
```

Comment:
Hm, it seems like we should be able to still run these tests against even deprecated functions. Maybe the right solution here is to suppress warnings about deprecated functions for this test only?